text/001-the-rfc-process.md

```diff
@@ -0,0 +1,220 @@
+  lodge any final objections before a decision is reached.
+- In most cases, the FCP period is quiet, and the RFC is either merged or
+  closed. However, sometimes substantial new arguments or ideas are raised,
+  the FCP is canceled, and the RFC goes back into development mode.
+
+## From RFC to implementation
+
+Once an RFC is merged it becomes "active" then authors may implement it and
+submit the feature as a pull request to the relevant repositories. Being
+"active" is not a rubber stamp, and in particular still does not mean the
+feature will ultimately be merged; it does mean that in principle all the major
+stakeholders have agreed to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "active" implies
+nothing about what priority is assigned to its implementation, nor does it imply
+anything about whether a developer has been assigned the task of implementing
+the feature. While it is not *necessary* that the author of the RFC also write
+the implementation, it is by far the most effective way to see an RFC through to
+completion: authors should not expect that other project developers will take on
+responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We strive
+to write each RFC in a manner that it will reflect the final design of the
+feature; but the nature of the process means that we cannot expect every merged
+RFC to actually reflect what the end result will be at the time of the next
+major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes should
+be new RFCs, with a note added to the original RFC.
+
+# Rationale and Alternatives
+[alternatives]: #rationale-and-alternatives
+
+The design space for decision making is very large, from democratic to
+autocratic and more.
+
+Forking and simplifying Rust's RFC process is *practical*. Rather than designing
+a decision making process from scratch, we take an existing one that works well
+and tailor it to our needs. Many Rust and WebAssembly stakeholders are already
+familiar with it.
+
+The main differences from the Rust RFC process are:
+
+- FCP lasts seven calendar days rather than ten. This reflects our desire for a
+  lighter-weight process that moves more quickly than Rust's RFC process.
+- The RFC template is shorter and merges together into single sections what were
+  distinct sections in the Rust RFC template. Again, this reflects our desire
+  for a lighter-weight process where we do not need to go into quite as much
+  painstaking detail as Rust RFCs sometimes do (perhaps excluding *this* RFC).
+
+The phases of RFC development and post-RFC implementation are largely the same
+as the Rust RFC process. I found that the motivations for nearly every phase of
```

Comment:
It might be better to remove uses of I in an RFC as it also serves as a reference if accepted. So it should be a bit more impersonal in nature.